bin/package_build.py

```diff
@@ -1,4 +1,4 @@
-#!/bin/python
+#!/usr/bin/python3
```

Comment:
Yeah, I had to make the same change on Debian. If this works better everywhere :+1: